src/client/components/mainFooter/mainFooter.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+import './mainFooter.css';
+
+class MainFooter extends React.Component {
```

Comment:
Same as in comment before, please also check your all components

Reply:
done)